implementations/micrometer-registry-azure-monitor/src/main/java/io/micrometer/azuremonitor/AzureMonitorMeterRegistry.java

```diff
@@ -113,13 +113,10 @@ protected void publish() {
     }
 
     private Stream<MetricTelemetry> trackLongTaskTimer(LongTaskTimer timer) {
-        MetricTelemetry active = createMetricTelemetry(timer, "active");
-        active.setValue(timer.activeTasks());
-
-        MetricTelemetry duration = createMetricTelemetry(timer, "duration");
-        duration.setValue(timer.duration(getBaseTimeUnit()));
-
-        return Stream.of(active, duration);
+        MetricTelemetry mt = createMetricTelemetry(timer, null);
```

Comment:
Not having any suffix seems problematic because how will you know it is a long task timer rather than a regular timer in the backend? Other than that issue, it seems like a reasonable change.

Reply:
How about adding the suffix `active` to the name to indicate it is a LongTaskTimer?